Index: src/DataProcess/ParseEmpData.scala
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/DataProcess/ParseEmpData.scala	(date 1572743248780)
+++ src/DataProcess/ParseEmpData.scala	(date 1572743248780)
@@ -26,7 +26,7 @@
     addItem(cItems, "788204", "Espresso", user)
     cItems("Espresso").price = -1
     cItems("Espresso").sku = ""                                                           //Done manually as there is no data for employee-only items.
-    cItems("Espresso").ingredients = List.empty
+    cItems("Espresso").ingredients = Array.empty
     cItems("Espresso").location = "Kept under the espresso machine, more is in the back on the coffee shelf."
     cItems("Espresso").nutrients = List.empty
 
